examples/chapter1/src/example3.cpp

```diff
@@ -53,11 +52,9 @@ int main() {
                               }
                               vectorPublisher->publish(vector);
                               std::this_thread::sleep_for(std::chrono::milliseconds(5));
-			      std::cout << "t (publisher) thread ID: " << std::this_thread::get_id() << std::endl;
+                              std::cout << "vectorPublisherThread thread ID: " << std::this_thread::get_id() << std::endl;
```

Comment:
Also rename the actual thread variable name (on line 47) from t to vectorPublisherThread

Reply:
Done